code/modules/power/singularity/generator.dm

```diff
@@ -10,6 +10,16 @@
 	var/energy = 0
 	var/creation_type = /obj/singularity
 
+	// You can buckle someone to the singularity generator, then start the engine. Fun!
+ 	can_buckle = 1

```

Comment:
Having checked out this branch, and looked at the field gens in the actual map, their can_buckle still appears to be set to 0. Same with some other objects. Maybe the map has them customized, or this is being overriden somewhere?

Reply:
i was spawning them in manually though so.

Reply:
Well, still worth ensuring the ones in Engineering have can_buckle = TRUE in practice. It would be weird if some let you, and some did not, with no obvious difference between them.

Reply:
well heres the thing the ones i spawn in have canbuckle false and still work so..i need to double check the one in engineering. I really DO NOT want to make a map edit on an item.

Reply:
also i can't set the 1/0 to true/false or it runtimes on me for the sing gen...wierd

Reply:
Was this resolved?

Reply:
The reason this won't compile is that there's a leading space hidden here
waaaagh why does it compile with 1s and 0s but not TRUE or FALSE